Src/Support/Google.Apis.Auth/OAuth2/UserCredential.cs

```diff
@@ -49,18 +59,43 @@ public TokenResponse Token
         /// <summary>Gets the user identity.</summary>
         public string UserId { get; }
 
+        /// <inheritdoc/>
+        public IReadOnlyDictionary<string, string> ExtraInformation { get; }
```

Comment:
nit: in general case, the http headers are a multi-map (same header can appear multiple times in the same request).

Reply:
Yep, good point.